core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -111,6 +111,15 @@ private TableProperties() {
   public static final String DELETE_AVRO_COMPRESSION = "write.delete.avro.compression-codec";
   public static final String AVRO_COMPRESSION_DEFAULT = "gzip";
 
+  public static final String PARQUET_BLOOM_FILTER_ENABLED = "write.parquet.bloom-filter-enabled";
+  public static final boolean PARQUET_BLOOM_FILTER_ENABLED_DEFAULT = false;
```

Comment:
What's the perf impact of writing bloom filer? Does it make sense to enable it by default? Would be nice to include benchmarks?

Reply:
Hi, Yufei, thanks for the review. The performance impact of writing bloom filter should be negligible, though we didn't do a performance benchmark. The cost of bloom filter is space. The default size of bloom filter for one column is 1 MB in each parquet file. If there are N columns in the table, then the extra space cost is **N MB in each file** to enable bloom filter for all columns. It is more reasonable to enable bloom filter only for columns that is of high cardinality and often used in filter expressions.

Reply:
Thanks for the explanation. Make sense to disable it by default. Is there any perf test from Parquet or Spark side we can refer to?

Reply:
Yes, I found [SPARK-35345](https://github.com/apache/spark/pull/32473) is adding perf test for Parquet bloom filter with Spark. And the author @huaxingao happens to be in this PR 😄

Reply:
I will update the Parquet bloom filter benchmark PR so it can be merged.